src/Views/ShortcutsView.vala

```diff
@@ -22,49 +22,47 @@ public class ShortcutOverlay.ShortcutsView : Gtk.Box {
     private const string SCHEMA_MUTTER = "org.gnome.mutter.keybindings";
 
     construct {
-        var windows_grid = new Gtk.Grid () {
-            column_spacing = 12,
-            hexpand = true,
-            row_spacing = 12
+        var windows_listbox = new Gtk.ListBox () {
+            selection_mode = BROWSE
```

Comment:
Is there any reason these listboxes are selectable? I feel this could be NONE.

![image](https://github.com/user-attachments/assets/facc58c7-61ec-429e-b6c4-6cc55844ba6b)

Reply:
When I tried that, the screen reader would then say "not selected" after reading each line which is really annoying 😅

Reply:
Ah I see. Is it possible to make selected rows as if not selected visually using CSS? If it's not possible or difficult let's just leave the reason why we don't use NONE here as a comment and resolve this conversation.

Reply:
Yeah can totally workaround this in CSS. Good call!